api/envoy/config/filter/http/header_to_metadata/v2/header_to_metadata.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.header_to_metadata.v2;
+option go_package = "v2";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Header-To-Metadata Fitlter]
+//
+// The configuration for transforming headers into metadata. This is useful
+// for matching load balancer subsets, logging, etc.
+//
+// :ref:`configuration overview <config_http_filters_header_to_metadata>`.
```

Comment:
This looks weird in the generated docs (https://65061-65214191-gh.circle-artifacts.com/0/source/generated/docs/api-v2/config/filter/http/header_to_metadata/v2/header_to_metadata.proto.html). Should probably have "Header-to-Metadata" at before the link so it reads "Header-to-metadata configuration overrview."